### PR TITLE
server: add unreachable store id

### PR DESCRIPTION
**Pull request by huachaohuang**

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

File: src/server/transport.rs [hunk 1]
```diff
@@ -132,8 +132,13 @@ impl RaftStoreRouter for ServerRaftStoreRouter {
...
 
-    fn report_unreachable(&self, region_id: u64, to_peer_id: u64) -> RaftStoreResult<()> {
-        RAFT_STORE_MSG_COUNTER.with_label_values(&["unreachable"]).inc();
+    fn report_unreachable(&self,
+                          region_id: u64,
+                          to_peer_id: u64,
+                          to_store_id: u64)
+                          -> RaftStoreResult<()> {
+        let store = to_store_id.to_string();
+        RAFT_STORE_MSG_COUNTER.with_label_values(&["unreachable", &*store]).inc();
```
Comment (siddontang) [inline src/server/transport.rs]: the metric has only one label. you can't use two values here.

Comment (huachaohuang) [inline src/server/transport.rs]: So, should we add store id to all `RAFT_STORE_MSG_COUNTER` labels?

**Comment by siddontang** [comment]
LGTM

PTAL @BusyJay @hhkbp2

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**